package-lock.json

```diff
@@ -32,7 +32,7 @@
       },
       "engines": {
         "homebridge": ">=1.3.2",
-        "node": ">=14.15.4"
+        "node": ">=14.16.0"
```

Comment:
Not sure why this one changed... is it still ok or should I undo this?

Reply:
your good! I can updated it.

Reply:
Ok, great. Ready for merge then, code still works fine for my MSL-420